bundle.Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
+
+FROM quay.io/operator-framework/operator-sdk:v1.13 as builder
+
+ARG release_tag
+# Copy the go source
+COPY . /workspace
+
+WORKDIR /workspace
+
+RUN microdnf install -y \
+      gcc \
+      git \
+    && microdnf clean all
```

Comment:
Nit: I don't think the clean is necessary for the builder. Feel free to ignore.

Reply:
Agree.  removed.